package.json

```diff
@@ -0,0 +1,8 @@
+{
+	"scripts": {
+		"format": "prettier -w **/*.js **/*.ts"
```

Comment:
lets drop adding a format package.json script and instead add rules_lint in a follow-up PR to run prettier

@mrmeku you can try to install it into this repo as a new rules_lint user and give Alex's docs a run: https://github.com/aspect-build/rules_lint

Reply:
Sounds good

Reply:
https://app.graphite.dev/github/pr/aspect-build/rules_cypress/78/chore-Integrate-rules_lint-and-format-files